packages/react/src/LogoSuite/LogoSuite.tsx

```diff
@@ -157,16 +157,26 @@ type LogoSuiteLogoBarProps = BaseProps<HTMLDivElement> & {
   /**
    * The stylistic variant of the LogoBar.
    */
-  variant?: 'muted' | 'emphasis'
+  variant?: 'muted' | 'emphasis' | 'default'
```

Comment:
Do we need 'default'? Given the color is dynamic and we only seem to alternate between muted and emphasis colors, can we drop this?

Reply:
On reflection potentially not, it doesn't do anything different to just leaving `variant` as `undefined`, so in hindsight it doesn't offer a lot of benefit.

Reply:
👍 cool, after seeing the related code, I think it behaves more like an `auto` variant, which better describes what it does.. but also agree it's probably better to just remove it altogether. Just for future reference, this is a good candidate for an quick API exploration (assuming it hasn't already happened)

Reply:
Yea that's worth keeping in mind in the future, will do, thanks 👍